app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -0,0 +1,177 @@
+class MiqCockpitWsWorker::Runner < MiqWorker::Runner
+  BINDING_ADDRESS = ENV['BINDING_ADDRESS'] || (Rails.env.production? ? nil : "0.0.0.0")
```

Comment:
Maybe in the future but it's fine for now.  This is very much like the one we use for the puma workers:  https://github.com/ManageIQ/manageiq/blob/616614ae7f9aac495a4f648adfd56f9ca3a09cee/app/models/mixins/miq_web_server_worker_mixin.rb#L5